src/main/java/com/garbagemule/MobArena/waves/MACreature.java

```diff
@@ -241,6 +263,12 @@ private static void registerTypeVariants() {
         //
         put("angrybee", "angrybees", "BEE", null);
         put("angrywolf", "angrywolves", "WOLF");
+        put("babypiglin", "babypiglins", "PIGLIN");
+        put("babyzombifiedpiglin", "babyzombifiedpiglins", "ZOMBIFIED_PIGLIN");
+        put("babyhoglin", "babyhoglins", "HOGLIN");
+        put("babyzoglin", "babyzoglins", "ZOGLIN");
+        put("babyhusk", "babyhusks", "HUSK");
+        put("babydrowned", "babydrowneds", "DROWNED");
```

Comment:
Let's go with `babydowned` as both singular and plural, similar to `explodingsheep`.

Reply:
Just pointing out the spelling mistake `downed` instead of `drowned` :)

Reply:
Yeah, wasn't sure which one to go with so I just added the s xD Will do